study_export.py

```diff
@@ -0,0 +1,104 @@
+import json
+from argparse import ArgumentParser
+from functools import cached_property
+
+import requests
+
+import psycopg2
+from ebi_eva_common_pyutils.config_utils import get_properties_from_xml_file
+from ebi_eva_common_pyutils.pg_utils import get_all_results_for_query
+
+
+class StudyExport:
+
+    def __init__(self, properties_file=None, stage='production'):
+        self.properties_file = properties_file
+        self.stage = stage
+
+    @cached_property
+    def connect(self):
+        properties = get_properties_from_xml_file(self.stage, self.properties_file)
+        pg_url = properties['eva.evapro.jdbc.url'].split('jdbc:')[-1]
+        pg_user = properties['eva.evapro.user']
+        pg_pass = properties['eva.evapro.password']
+        conn = psycopg2.connect(pg_url, user=pg_user, password=pg_pass)
+        return conn
```

Comment:
If you upgrade to pyutils >= 0.3.15 (I think), you can use `get_metadata_connection_handle`

Reply:
that's right. I've added it.